.jenkins/Jenkinsfile

```diff
@@ -27,18 +28,55 @@ pipeline {
             ssh-keyscan -t rsa github.com > /root/.ssh/known_hosts
 
             bundle install
+            bundle exec appraisal install
           '''
         }
       }
     }
-    stage('Unit Test') {
-      steps {
-        script {
-          sh 'bundle exec rake'
+    stage('Appraisals') {
+      parallel {
+        stage('Current') {
```

Comment:
Interesting. When will current be different from Rails 4?

Reply:
I think it already is due to the dependencies in the `gemspec` needing to be opened up in order to get appraisal to be able to install different versions of the rails gems.  I think the current `Gemfile.lock` right now is testing against rails 6.

Reply:
Ok.